x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/search_index_selectable.tsx

```diff
@@ -44,8 +45,11 @@ export interface SearchIndexSelectableOption {
 
 const healthColorsMap = {
   red: 'danger',
+  RED: 'danger',
```

Comment:
question: why do we have both lowercase and uppercase versions here? Is it possible to stick to just one of them?

Reply:
The `HealthStatus` can be both uppercase and lowercase colors. If I try to have one version and try to map them here with JS way it turns the value into string, so I have to cast it to the `HealthStatus`. 

```typescript
healthColorsMap[option.health.toLowercase() as HealthStatus]
```

If there is a better way to do that I am not aware of it, but open to suggestions.

Reply:
We have complete control over these values, right? I don't see why we can't be consistent everywhere 🤔

Reply:
I mean we should not be doing .toLowerCase() (or .toUpperCase()) and casting. We should just be consistent and use either only uppercase or only lowercase everywhere. Unless I'm missing something here?

Reply:
So the values are provided from the elasticsearch query, they could be both. Only other possibility is changing backend response on our side, to either one of the values. If we are going to do that I would do it as an incrementation as this PR waited quite some time.

Reply:
We can skip this for now, but we should never really be in a situation where an enum can have values like this, we need to unify that somewhere well before we hit this. 

I'm also not sure why the query could return either? Looking at the documentation the query itself should only ever return lowercase values.

Reply:
very rude! haha